src/app/pages/send/send-crypto-asset-form/components/selected-asset.tsx

```diff
@@ -0,0 +1,24 @@
+import { Text } from '@stacks/ui';
+
+import { SelectedAssetField } from '@app/components/forms/selected-asset-field';
+import { Flag } from '@app/components/layout/flag';
+import { SpaceBetween } from '@app/components/layout/space-between';
+
+interface SelectedAssetProps {
+  icon: React.JSX.Element;
+  name: string;
+  symbol: string;
+}
+export function SelectedAsset({ icon, name, symbol }: SelectedAssetProps) {
+  return (
+    <SelectedAssetField mb="base" name="symbol">
+      <Flag img={icon} align="middle" spacing="tight">
+        <SpaceBetween>
+          <Text>
+            {name} ({symbol.toUpperCase()})
```

Comment:
I think we already pass upper-case symbols and we could also use a `<span` here with CSS - `text-transform: uppercase;` rather than running a function

Reply:
I actually just renamed this file.